scripts/__input_system_tick/__input_system_tick.gml

```diff
@@ -166,9 +159,19 @@ function __input_system_tick()
         ++_m;
     }
     
-    __input_release_multimonitor_cursor();
+    //Block mouse capture when the window state changes
+    if (_global.__mouse_capture && __input_window_changed())
```

Comment:
Should this be `... && _global.__window_focus && __input_window_changed()`? I'm concerned about window state changing whilst the window is out of focus leading to `__input_window_changed()` returning `false` when the window is back in focus.